### PR TITLE
Removed timestamp arguement from avatarURL

### LINKED ISSUE #1587: Avatar Re renders onScroll everytime
**Actual Behaviour**

Avatar updates on every event.

susis2

**Expected Behaviour**

It should load for once until you refresh.

susiski

**Steps to reproduce it**

Go to https://skills.susi.ai/ then log in and scroll

**Would you like to work on the issue?**

Yes.

### PR DESCRIPTION
**Pull request by aashutoshrathi**
Fixes #1587 

Changes: Removed timestamp URL from avatar fetch as it was triggering auto-fetch avatar everytime some event occurs.

Surge Deployment Link: https://pr-1588-fossasia-susi-skill-cms.surge.sh

Screenshots for the change:

susiski

### CODE REVIEW

**Comment by aashutoshrathi** [comment]
@akshatnitd I think loading image every time an event occurs is a way too heavy thing to do for image update.
A page refresh would be more preferred.

**Comment by aashutoshrathi** [comment]
@akshatnitd I agree.
I am on it.

**Comment by aashutoshrathi** [comment]
@akshatnitd Check now, I fixed it so that random query number remains same until someone refresh page.

File: src/components/StaticAppBar/StaticAppBar.react.js [hunk 1]
```diff
@@ -53,6 +53,7 @@ class StaticAppBar extends Component {
       showOptions: false,
       showAdmin: false,
       anchorEl: null,
+      num: new Date().getTime(),
```
Comment (akshatnitd) [inline src/components/StaticAppBar/StaticAppBar.react.js]: Please name it to `timestamp`, `num` is unclear.

Comment (aashutoshrathi) [reply src/components/StaticAppBar/StaticAppBar.react.js]: Sure, on it.

**Comment by aashutoshrathi** [comment]
Squashed @akshatnitd

**Pull request merged**